posts/github-intro-tvaers-2025/slides/github-intro-presentation.qmd

```diff
@@ -0,0 +1,155 @@
+---
+title: "Introduction to GitHub"
+subtitle: "Tværs Gruppemøde, April 2025"
+author: "Signe Kirk Brødbæk"
+format: 
+    revealjs:
```

Comment:
```suggestion
    revealjs:
        theme: default
```

This forces it to use a revealjs theme rather than the Seedcase theme, so that it gets rendered nicely from the suggestion I made above.